sdv/data_processing/data_processor.py

```diff
@@ -10,6 +10,7 @@
 import rdt
 from pandas.api.types import is_float_dtype, is_integer_dtype
 from rdt.transformers import AnonymizedFaker, IDGenerator, RegexGenerator, get_default_transformers
+from rdt.transformers.pii.anonymization import get_anonymized_transformer
```

Comment:
maybe we can move `get_anonymized_transformer` to the `__init__` of `rdt.transformer.pii` and import it directly from there.

Reply:
That seems reasonable as well but I think the subname `anonymization` gives context to the library. (This change would be in RDT)

Reply:
Makes sense 👍🏻